.github/workflows/publish_next_compute-baseline.yml

```diff
@@ -7,6 +7,8 @@ on:
     paths:
       - packages/compute-baseline/**
 
+permissions: {}
+
```

Comment:
The permissions vary between jobs here, so this explicitly unsets them for the workflow.